Eukalyptus-Dreizack/deepcity-framework
# Add minimal prettier configuration

## Description
Set the minimal required prettier configuration. Added the appropriate packages as dev dependencies. Updated eslint to the latest version.

## Related issue
#59

## How can this be tested?
Use single quotes and save your changes. The single quotes should be converted to double quotes.

## Motivation and context
Ease the develoment by automatically fixing small and tedious things.

## Additional context
Enable the `formatOnSave` setting in vscode.
